### PR TITLE
New community page

### PR DESCRIPTION
**Pull request by lmarini**

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Community
+description: How to contribute to the Clowder community.
+background: '/PATH_TO_IMAGE'
```
Comment (robkooper) [inline community.md:5]: should this be here?

File: community.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+There are many ways to get involved in the Clowder community. Below we have identified a few ways and pointers to get started. *The easiest way to get more information is to join us on [Slack][slack].* Before participating, please take a momente to [read our Code of Conduct](https://github.com/clowder-framework/clowder/blob/develop/CODE_OF_CONDUCT.md).
```
Comment (robkooper) [inline community.md:8]: do we want to point to the master branch for conduct?

Comment (robkooper) [inline community.md:8]: missing url for slack

Comment (robkooper) [reply community.md:8]: not sure never seen the reference to footnotes.

**Review by robkooper — CHANGES_REQUESTED** [review]
looks good, small questions

File: community.md (hunk 1 above)
Comment (lmarini) [reply community.md:5]: It's a placeholder. I am trying to figure out what to put there to make the header more interesting and still readeable but I am not sure what. For some reason it doesn't complaint.

File: community.md (hunk 2 above)
Comment (lmarini) [reply community.md:8]: Thanks. I really think we should change the default to `master`. This keeps happening to me. Even when I remember I still have to manually change the branch. With our current workflow I think having master would be best and easier.

Comment (lmarini) [reply community.md:8]: yeah, it's in the footnote. Was thinking of linking to slack everywhere.

**Pull request merged**